src/LocalStorageStore.ts

```diff
@@ -0,0 +1,54 @@
+import Store from "./Store";
+import MemoryStore from "./MemoryStore";
+
+/** Class that stores key/value pairs in localStorage if possible
+    but falls back to an in-memory store. */
+export default class LocalStorageStore implements Store {
+    private enabled: boolean;
```

Comment:
It seems like `enabled` is really `isUsingLocalStorage`. Perhaps this can be better named.